### PR TITLE
Bah 781 choice bug

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Created unique upload question property name for the 781a and 781.

## Testing done
-[x] Test against original defect steps to reproduce

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by aaronjlech — APPROVED** [review]
LGTM!

File: src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx [hunk 1]
```diff
@@ -423,17 +424,31 @@ describe('526 helpers', () => {
...
       };
-      expect(isUploadingPtsdForm(formData)).to.be.true;
+      expect(isUploading781Form(formData)).to.be.true;
     });
 
-    it('should return false if user has not chosen to upload documents', () => {
+    it('should return false if user has not chosen to upload 781', () => {
       const formData = {};
-      expect(needsToEnter781({ formData })).to.be.false;
+      expect(isUploading781Form({ formData })).to.be.false;
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx]: I hate to be a pain--and I know this wasn't introduced in this PR--but this shouldn't be destructured. :sweat_smile:

File: src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx [hunk 2]
```diff
@@ -423,17 +424,31 @@ describe('526 helpers', () => {
...
+    it('should return true if user has chosen to upload 781a', () => {
+      const formData = {
+        'view:upload781aChoice': 'upload',
+      };
+      expect(isUploading781aForm(formData)).to.be.true;
+    });
+
+    it('should return false if user has not chosen to upload 781a', () => {
+      const formData = {};
+      expect(isUploading781aForm({ formData })).to.be.false;
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx]: This one too.

**Review by cvalarida — COMMENTED** [review]
Looks good! The unit tests still need to be fixed up, it looks like, but it looks good. :smile:

**Review by markgreenburg — APPROVED** [review]
Looks good!

**Review by cvalarida — APPROVED** [review]
Excellent!

**Pull request merged**